style/configs/config.planetiler-sf.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+/*
+Planetiler tile server, hosted by StreetFerret
+*/
+const TILESOURCE = [
+  "http://vtile.streetferret.com:8080/data/v3/{z}/{x}/{y}.pbf",
+];
```

Comment:
Why a direct pbf url here?  I expected `http://vtile.streetferret.com:8080/data/v3.json`

Reply:
Actually that one can be.  I've having trouble getting the tileserver to respond on the domain name it came from.  The bigger issue is the HTTPS proxied version, e.g. take a look at the pbf url present in: https://6ug7hetxl9.execute-api.us-east-2.amazonaws.com/data/v3.json

By supporting both a direct tile source as well as a JSON config, you have the possibility of specifying either.

Reply:
See f882610 which is part of my motivation for this PR - getting a working https-backed tile source that we can deploy.

Reply:
Does the direct tile source behave differently than the JSON config?  Is there a need for it?

Reply:
As far as I can tell, the only thing the JSON config provides of value is the pbf direct link, the OpenMapTiles attribution (which I have just hard-coded in by default now), and a bunch of layer definition stuff that we don't use.

Reply:
> Actually that one can be. I've having trouble getting the tileserver to respond on the domain name it came from.

Are you passing `--public_url https://6ug7hetxl9.execute-api.us-east-2.amazonaws.com/` to tileserver?

It seems to respect Host but not X-Forwarded-Host, so maybe you can ask your Lambda to send a custom host header instead.

Reply:
Oh yeah now we're cooking with gas.